This requires the user to be in the group used for ydotoold, rather
than the same user. This means a user in the correct group can use the
service without root permissions.

The socket path is changed due to the restricted deletion flag of /tmp,
which stops a member of the same group overwriting it, which ydotoold
does each time.

diff --git a/Client/ydotool.c b/Client/ydotool.c
index c86f20b..79bd8b0 100644
--- a/Client/ydotool.c
+++ b/Client/ydotool.c
@@ -134,7 +134,7 @@ int main(int argc, char **argv) {
 	if (env_sp) {
 		daemon_socket_path = env_sp;
 	} else {
-		daemon_socket_path = "/tmp/.ydotool_socket";
+		daemon_socket_path = "/var/lib/ydotoold/socket";
 	}
 
 	fd_daemon_socket = socket(AF_UNIX, SOCK_DGRAM, 0);
diff --git a/Daemon/ydotoold.c b/Daemon/ydotoold.c
index 10b4ca2..e0d061b 100644
--- a/Daemon/ydotoold.c
+++ b/Daemon/ydotoold.c
@@ -59,8 +59,8 @@
 
 #include <linux/uinput.h>
 
-static const char *opt_socket_path = "/tmp/.ydotool_socket";
-static const char *opt_socket_perm = "0600";
+static const char *opt_socket_path = "/var/lib/ydotoold/socket";
+static const char *opt_socket_perm = "0660";
 
 static void show_help() {
 	puts(
